docs/gen-ai/gen-ai-metrics.md

```diff
@@ -0,0 +1,157 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI metrics
+--->
+
+# Semantic Conventions for Generative AI Metrics
```

Comment:
```suggestion
# Semantic Conventions for Generative AI Client Metrics
```

Reply:
Since we differentiate between `request`/`response` (physical) and `operation` (logical), we shouldn't differentiate between client and server here. The `operation` consumed a number of tokens and took a certain duration of time regardless of whether it happened on a client (local model) or a server (hosted model) or whether the client or server is reporting the telemetry.

Reply:
If I use the same name for client and server (GenAI system) metrics:
- it will have to have the same set of attributes with the same requirement levels. I don't think it's realistic to expect. The GenAI server metrics (if ever reported) would likely have different set of attributes
- Users who see server and client in the same metric, would have trouble figuring out which one is which. 
- The client and server values will be different. E.g. when I send a request and don't get response due to the network issue, the client metric won't get the usage number for that request.